log/logger.go

```diff
@@ -57,6 +57,8 @@ type Logger struct {
 	mutex        *sync.Mutex
 }
 
+var pid = os.Getpid()
```

Comment:
not sure if there any race since this is declared global

Reply:
I can't figure out the possible race from my experience, I think pid in a process's lifecycle is always same one, it only need init once.

Anyway I'm glad to adjust the code to your perfer style.

Reply:
yes you are right.  shouldn't be a problem